core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -769,6 +778,28 @@ class HttpGoogleServicesDAO(
         case t: HttpResponseException if t.getStatusCode == StatusCodes.NotFound.intValue => None
       }
     }
+
+    def lifeSciencesBetaHandler(opId: String) = {
+      val lifeSciencesAccountCredential = getLifeSciencesServiceAccountCredential()
+      lifeSciencesAccountCredential.refreshToken()
+      val lifeSciencesApi = new CloudLifeSciences.Builder(httpTransport, jsonFactory, lifeSciencesAccountCredential).setApplicationName(appName).build()
+      val operationRequest = lifeSciencesApi.projects().locations().operations().get(opId)
+      implicit val service = GoogleInstrumentedService.LifeSciences
+
+      retryWithRecoverWhen500orGoogleError(() => {
+        // Google library returns a Map[String,AnyRef], but we don't care about understanding the response
+        // So, use Google's functionality to get the json string, then parse it back into a generic json object
+        Option(executeGoogleRequest(operationRequest).toPrettyString.parseJson.asJsObject)
+      }) {
+        // Recover from Google 404 errors because it's an expected return status.
+        // Here we use `None` to represent a 404 from Google.
+        case t: HttpResponseException if t.getStatusCode == StatusCodes.NotFound.intValue => None
+      }
```

Comment:
if 789-797 is a copy/paste from above can this get pulled out to a function and shared between the v2s?

Reply:
it's not _quite_ a copy/paste. One uses a Genomics object and the other a LifeSciencesCloud object.

I did consider making a "can be queried for operation details" typeclass and genericizing this function... but then I just copy/pasted instead